docs/development/releasing.rst

```diff
@@ -274,30 +277,43 @@ The procedure for this is straightforward:
 
       $ git branch v<version>.x
 
-#. Render the changelog with towncrier, as above.
```

Comment:
As @Cadair mentioned, we (a) only want to do this for the final release (not at branching time or RCs etc), and (b) we should do it on the release branch and forward-port it to ``main``

Reply:
Hmm I agree with (b) but not necessarily (a).  See my suggestion just below